odoo/addons/base/tests/test_qweb.py

```diff
@@ -1240,6 +1241,51 @@ def test_error_message_2(self):
             self.assertIn('Can not compile expression', error)
             self.assertIn('<div t-esc="abc + def + ("/>', error)
 
+    def test_error_message_3(self):
+        # The format error tells the developer what to do.
+        template = '''<section>
+                    <div t-esc="abc + def + (">
+                        <span>content</span>
+                    </div>
```

Comment:
wouldn't it be better to have a "well formatted" template ? (here the syntax is not correct)
```
abc + def + (
```

maybe just something like this ?
```
1+2
```